astropy_sphinx_theme/bootstrap-astropy/static/bootstrap-astropy.css

```diff
@@ -317,7 +317,7 @@ div.dropdown-content {
     background-color: DimGray;
     color: White;
     width: 235px;
-    height: 155px;
+    height: 125px;
```

Comment:
what about using the same numbers as in the main webpage? 
I believe these are the relevant ones (but please double check whether it's indeed the css that's being used on astropy.org: https://github.com/astropy/astropy.github.com/blob/master/css/style.css#L651)

Reply:
Yeah, that's what I did originally. But for some reasons things didn't work out then if I use their settings as follows:
```
.dropdown-content {
...
min-width: 160px;
width: 100%;
height: 100%
...
}
```
I cannot recall why their code did not work for this case though. But if I use 100% width, I remember getting the full length of the screen instead of that of the individual menu item. (Similarly, if I use 100% for the height, it got rendered to the full height of the screen when I was testing the code, which is not really desirable.)Further, I do know that I have to add in css styling for a new division (.div) called `topbar` for things to work in my case, which does not exist in their code. I basically reworked everything to get the top menubar working for the docs.

Reply:
ok, never mind it then (I didn't follow everything in this PR).

Reply:
No worries. I had to resort to using .div to make the menubar work, maybe that's why their code doesn't work on this case.